.goreleaser.yaml

```diff
@@ -7,6 +7,9 @@ builds:
   - env:
       - CGO_ENABLED=0
     dir: ./cmd/calendarsync
+    ldflags:
+      - -X 'main.BuildTime={{.Date}}'
```

Comment:
`.Date` looks like it would result in non-reproducible binaries? What about `.CommitDate` instead (or just removing the BuildTime completely?) We can also defer that discussion to an issue.